systemtest/source_tests/test_source_1/runtest_source_1.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python3
```

Comment:
What do you test here? Could you write a readme at the top of the file? That would it make easier to review.

Reply:
I think this is now solved.